templates/tutorialv2/includes/content/warn_typo.part.html

```diff
@@ -14,6 +14,18 @@
         </a>
 
         {% crispy form_warn_typo %}
-    </div>
+        <a href="#report-obsolete" class="open-modal btn btn-grey ico-after edit blue">
```

Comment:
```suggestion
        <a href="#report-obsolete-modal" class="open-modal btn btn-grey ico-after edit blue">
```
(par cohérence avec ce qui est fait pour signaler une faute)